srcpkgs/ngspice/template

```diff
@@ -13,7 +12,7 @@ maintainer="Toyam Cox <Vaelatern@voidlinux.org>"
 license="BSD-3-Clause"
 homepage="http://ngspice.sourceforge.net"
```

Comment:
```suggestion
homepage="https://ngspice.sourceforge.net"
changelog="https://sourceforge.net/p/ngspice/ngspice/ci/master/tree/NEWS?format=raw"
```

Reply:
Thanks! I added that changelog.  Looks like the homepage is now at sourceforge.io.